PKGBUILD

```diff
@@ -14,8 +14,8 @@ build() {
 }
 
 package() {
-    cd DollarSkip-${pkgver} &&
-    install -m 755 -D temp "${pkgdir}/usr/local/bin/\$" 
+    cd DollarSkip-${pkgver}
+    make install DESTDIR="${pkgdir}"
```

Comment:
The AUR will need to be different, since this installs to /usr/local/bin

Reply:
So we can add `PREFIX="/usr"` to the makefile for the aur.